assets/app/view/invite_game.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'game_manager'
+require 'view/game_row'
+
+module View
+  class InviteGame < Snabberb::Component
+    include GameManager
+
+    needs :user
+    needs :refreshing, default: nil, store: true
+
+    def render
+      children = [h(:h2, {}, 'This game has not started yet')]
+      if !@user
+        children << h(:h3, {}, 'You need to login before joining a game:')
+        children << h(View::User, type: :login)
+      else
+        children << h(:div, {}, [h(GameCard, gdata: @game, user: @user)])
+      end
+
+      children << h(:a, { attrs: { href: '/' } }, 'Return home')
```

Comment:
you don't need this, you can click on the banner

Reply:
I usually click on banner, but I wasn't sure if the average user was as comfortable with that. I'll remove it.